eduNEXT/xblock-mindmap
# feat: show assignment information to students

### Description
This PR adds:
- Information on the assignment in the LMS.
- The list of submissions only shows those in `submitted` and `completed` status.

### How To Test
1. Install this branch in your environment.
2. Add a block in a unit from Studio.
3. As a student, you should be able to see your assignment information.
4. As an instructor, you should only see submissions in `submitted` and `completed` status.

### Screenshots
#### As Student
- Not Attempted
![image](https://github.com/eduNEXT/xblock-mindmap/assets/64033729/25f60387-e0fb-4937-9535-330cbeca9d34)
- Submitted
![image](https://github.com/eduNEXT/xblock-mindmap/assets/64033729/cc060f3a-0dea-44d3-907d-9ababb80c4a1)
- Completed
![image](https://github.com/eduNEXT/xblock-mindmap/assets/64033729/da424286-c906-4a71-8c1b-1c5d80cbc4f8)

#### As Instructor
![image](https://github.com/eduNEXT/xblock-mindmap/assets/64033729/b0cc0bf5-38e4-4d79-a626-32410252c365)
